tests/functional/platforms/03-platform-db/flow.cylc

```diff
@@ -0,0 +1,10 @@
+[scheduling]
+    [[graph]]
+        R1 = frozen & disney
+[runtime]
+    [[frozen]]
+        script = "echo let it go"
+        platform = elsa
+    [[disney]]
+        script = "echo cant hold it back anymore"
+        platform = olaf
```

Comment:
What if we get sued by Disney? 😆

Reply:
No, since it's obviously a ref to Olaf II of Norway... ;p

Reply:
Also a missed opportunity to wind Oliver up with an apostrophe in "can't".